jinja2cli/cli.py

```diff
@@ -197,6 +197,12 @@ def cli(opts, args):
             ext = 'jinja2.ext.' + ext
         extensions.append(ext)
 
+    # Use only a specific section if needed
+    if opts.section:
+        section = opts.section
+        if section in data:
```

Comment:
imo, I'd want to throw the `KeyError` here.

Since we're explicitly saying "I want to use this section", but that section doesn't exist in the config. I'd expect it to error.